logviewer/src/main/java/com/nodesagency/logviewer/screens/logdetails/LogDetailsViewModel.kt

```diff
@@ -0,0 +1,24 @@
+package com.nodesagency.logviewer.screens.logdetails
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.nodesagency.logviewer.data.LogRepository
+import com.nodesagency.logviewer.data.model.LogDetails
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+
+internal class  LogDetailsViewModel(private val logRepository: LogRepository, logEntryId: Long) : ViewModel() {
+
+    private val mLogEntryLiveData : MutableLiveData<LogDetails> = MutableLiveData()
```

Comment:
Could you replace the m-prefixed member name with `logEntryLiveData` please, so it stays consistent with the naming in other places?

Reply:
Fixed :)